README.md

```diff
@@ -78,23 +81,48 @@ require("search").setup({
   -- its also possible to overwrite the default tabs using the tabs key instead of append_tabs
   tabs = {
     {
-	  "Files",
-	  function(opts)
-	    opts = opts or {}
-		if vim.fn.isdirectory(".git") == 1 then
-		  builtin.git_files(opts)
-		else
-		  builtin.find_files(opts)
-		end
-	  end
+      "Files",
+      function(opts)
+        opts = opts or {}
+        if vim.fn.isdirectory(".git") == 1 then
+          builtin.git_files(opts)
+        else
+          builtin.find_files(opts)
+        end
+      end
     }
-  }
+  },
 })
 ```
 
+### Tab Collections
+If you want to group certain pickers together into separate search windows you can use the collections keyword:
+
+```lua
+local builtin = require('telescope.builtin')
+require("search").setup({
+  tabs = {
+    initial_tab = 1,
+    tabs = { ... } -- As shown above
+  },
+  collections = {
+    -- Here the "git" collection is defined. It follows the same configuraton layout as tabs.
+    git = {
+      initial_tab = 1, -- Git branches
+      tabs = {
+        { name = "Branches", tele_func = builtin.git_branches },
+        { name = "Commits", tele_func = builtin.git_commits },
+        { name = "Stashes", tele_func = builtin.git_stash },
+      }
+    }
+  }
+})
+``` 
+
 ### known issues
 - pickers with more-than-average loading time (like lsp related, or http sending pickers) can feel a bit off, since the UI will wait for them to be ready.
 - heavily custom configured telescope settings (like in many nvim distros) might lead to unexpected errors, please open an issue if you encounter any.
+- A window with no available pickers can cause neovim to hang.
```

Comment:
could you provide a minimal config for this? in my case - if I add a collection / default list with only broken telescope pickers I get a stack overflow.. is that the issue or is it something different?

Reply:
It is a stack overflow at least. What I did was add a collection with only git pickers, with the available func from the readme
```lua
      local is_git_dir = function ()
        return vim.fn.isdirectory(".git") == 1
      end
      
      -- other config
        collections = {
          git = {
            initial_tab = 1, -- Git files
            tabs = {
              { name = "Git files", tele_func = telescope.git_files, available = is_git_dir },
              { name = "Branches", tele_func = telescope.git_branches, available = is_git_dir },
              { name = "Commits", tele_func = telescope.git_commits, available = is_git_dir },
              { name = "Status", tele_func = telescope.git_status, available = is_git_dir },
              { name = "Stash", tele_func = telescope.git_stash, available = is_git_dir },
            }
          },
        }

```
If I'm not in a git directory the stack will overflow.

Reply:
ah okay I, i'll add this as an issue :)